parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReaders.java

```diff
@@ -168,6 +172,27 @@ public void setPageSource(PageReadStore pageStore, long rowPosition) {
     }
   }
 
+  static class DeleteMarkerReader implements ParquetValueReader<Boolean> {
```

Comment:
Isn't there a constant reader that we can reuse?

Reply:
Yes, we could use the existing constant reader.